test/Incremental/Inputs/superfluous-cascade/definesPoint-after.swift

```diff
@@ -0,0 +1,4 @@
+struct Point {
+  let x: Float
+  let y: Float
```

Comment:
I had expected `let x, y: Int` instead of these. Just curious why you chose this change?

Reply:
I guess it's a little bit better to change from Float to Int, because a sufficiently smart compiler might recognize the current change as a no-op.

Reply:
Good point. I've revised it to go from Float to Double.